src/App.js

```diff
@@ -12,13 +12,80 @@ const goodsFromServer = [
   'Honey',
   'Jam',
   'Garlic',
-];
-
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+].map((good, index) => ({
+  product: good,
+  id: index + 1,
+}));
```

Comment:
Якщо тобі потрібен масив в такому форматі, тоді краще створити для цього окрему змінну і використовувати її.
```suggestion
]

const preparedGoods = goodsFromServer.map((good, index) => ({
  product: good,
  id: index + 1,
}));
```

Reply:
спасибо большое, хорошие правки, исправила!)